lang/en/quizaccess_seb.php

```diff
@@ -123,6 +127,7 @@
 $string['fileparsefailed'] = 'The uploaded file could not be saved as a SEB config file.';
 $string['clientrequiresseb'] = 'This quiz has been configured to use the Safe Exam Browser with client configuration.';
 $string['settingsfrozen'] = 'Due to there being at least one quiz attempt, the Safe Exam Browser settings can no longer be updated.';
+$string['passwordnotset'] = 'A setting has been enabled which requires all quizzes to have a password set.';
```

Comment:
I'd suggest to change this string to clearly explain that the password is required, because of using SEB